### PR TITLE
Node 20 🎉

### PR DESCRIPTION
**Pull request by GGKapanadze**
# Description

After the maestro's introduction, we can upgrade from `node:16` to `node:20`. This allows us to remove extra `prep.` scripts during the installation. PR also updates the version of node for the github workflows

| Command | Tested |
|--------|--------|
| `pnpm build:libs`  | ✅  |
| `pnpm twig:dev:docs` | ✅ |
| `pnpm twig:build:docs`  | ✅ |
| `pnpm react:dev:docs` | ✅ | 
| `pnpm react:build:docs` | ✅ | 

> `pnpm build:libs` covers all internals

### CODE REVIEW

File: .nvmrc [hunk 1]
```diff
@@ -1 +1 @@
-v20.11.1
+v20
```
Comment (bashlk) [inline .nvmrc:1]: Just to be safe, might be better to set this to the same exact version as the workflows

File: package.json [hunk 2]
```diff
@@ -27,7 +27,8 @@
     }
   ],
   "engines": {
-    "pnpm": "8.6.0"
+    "pnpm": "8.6.0",
+    "node": ">=20"
```
Comment (bashlk) [inline package.json:31]: Same note here

File: .nvmrc (hunk 1 above)
Comment (GGKapanadze) [reply .nvmrc:1]: Do you want to freeze `patch` versions, too?

Comment (bashlk) [reply .nvmrc:1]: I think so, so as to keep it consistent. Then a single search for the version number will also reveal all the places in which it has to be updated.

**Review by bashlk — APPROVED** [review]

**Pull request merged**